HousingRepairsOnlineApi/Helpers/NotificationConfiguration/CommunalNotificationConfigurationProvider.cs

```diff
@@ -51,7 +46,7 @@ public class CommunalNotificationConfigurationProvider : INotificationConfigurat
         };
     }
 
-    public Dictionary<string, dynamic> GetPersonalisationForSMSTemplate(Repair repair)
+    public Dictionary<string, dynamic> GetPersonalisationForSmsTemplate(Repair repair)
```

Comment:
Use interface for return type
```suggestion
    public IDictionary<string, dynamic> GetPersonalisationForSmsTemplate(Repair repair)
```